src/main/java/com/ecwid/consul/v1/QueryParams.java

```diff
@@ -47,6 +47,13 @@ public QueryParams(long waitTime, long index) {
 		this.index = index;
 	}
 
+    public QueryParams(String datacenter, long waitTime, long index) {
```

Comment:
Why not go with a builder pattern since every combination of fields would be unruly.